lib/test_kfence.c

```diff
@@ -31,11 +31,31 @@ int do_test(size_t size)
 	return 0;
 }
 
+int do_test_uaf(size_t size)
```

Comment:
Should these be static?

Not that it matters, since we'll rewrite this for kunit anyway (I'm happy to rewrite to kunit, so no action required by you).

Reply:
Thanks!
I'll add static anyway.